extensions/notebook/src/protocol/notebookUriHandler.ts

```diff
@@ -33,15 +32,25 @@ export class NotebookUriHandler implements vscode.UriHandler {
 				vscode.window.showErrorMessage(localize('notebook.unsupportedAction', "Action {0} is not supported for this handler", uri.path));
 		}
 	}
-
+	/**
+	 * Our Azuredatastudio URIs follow the standard URI format of
+	 * azuredatastudio://microsoft.notebook/open?url=...
```

Comment:
This should really : 

1. Give examples of actual full URIs
2. Mention that it supports parameters as well and show examples of that
3. Mention that it only supports http(s) links

Reply:
When writing comments always try to approach it from the point of view of someone who isn't intimately familiar with the code - and so outlining specific formats expected/supported is key to making it easy for other people to quickly and easily understand what is supposed to be having. (with tests backing it up to ensure that functionality is being kept the same)

Reply:
Totally agree. It is quite interesting to me that there were about zero comments from when I started this, was this never recommended earlier on? I think it is a fantastic idea to give thorough comments, as this will help engineers of all types.